modules/ROOT/pages/change-jdk-config-in-projects.adoc

```diff
@@ -5,18 +5,20 @@ endif::[]
 
 You can select any of your installed JREs as the default Java runtime environment that Studio uses to run your Mule project.
 
-You can change your JRE if you have a specific JDK of Java 11 that you must use or to downgrade to JDK 1.8 to run applications in Mule runtime engine 4.1.x and earlier.
+You can change your JRE to use a specific JDK of Java, or to downgrade to JDK 1.8 to run applications on Mule runtime engine 4.1.x and earlier.
```

Comment:
@tomasperezc, please, can you take a look at this sentence and confirm that it is correct?
Thanks!

Reply:
Hi, the selected JDK for a project it is used beyond custom code... it is used to to run/debug the application and that it means that you will need a compatible JDK with the Runtime version of the project.
Regarding Custom code, it will depend also on the configuration of `maven-compiler-plugin` in the pom.xml and also the features used in those `.java` files and libs.

What I find strange here is that we mention 4.1.x and earlier when they are way past the Extended Support date (Nov 2022 for 4.1).... and earlier would mean 3.9.x which is not supported in Studio 7:
https://www.mulesoft.com/legal/versioning-back-support-policy#support-matrix

Reply:
After chatting with Valkyrie and Martín, we decided to remove the Mule runtime 4.1 and SDK 1.8 reference.